features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -1225,30 +1222,41 @@ void ATHandler::flush()
     }
 }
 
-void ATHandler::debug_print(const char *p, int len)
+void ATHandler::debug_print(const char *p, int len, ATType type)
```

Comment:
I really don't like that we have this precedent of internal/external APIs, and which are alright to change in a patch, but so it goes.

Reply:
Especially since this a debug function.